src/tlo/methods/rti.py

```diff
@@ -4131,12 +4147,13 @@ def apply(self, person_id, squeeze_factor):
             if code[0] in df.loc[person_id, 'rt_injuries_for_open_fracture_treatment']:
                 df.loc[person_id, 'rt_injuries_for_open_fracture_treatment'].remove(code[0])
         else:
-            self.sim.modules['RTI'].schedule_hsi_event_for_tomorrow(self)
-            if pd.isnull(df.loc[person_id, 'rt_date_death_no_med']):
-                df.loc[person_id, 'rt_date_death_no_med'] = self.sim.date + DateOffset(days=7)
-            logger.debug(key='rti_general_message',
-                         data=f"Person {person_id}'s has {open_fracture_counts} open fractures without treatment",
-                         )
+            if self._number_of_times_this_event_has_run < self._maximum_number_times_event_should_run:
+                self.sim.modules['RTI'].schedule_hsi_event_for_tomorrow(self)
+                if pd.isnull(df.loc[person_id, 'rt_date_death_no_med']):
+                    df.loc[person_id, 'rt_date_death_no_med'] = self.sim.date + DateOffset(days=7)
+                logger.debug(key='rti_general_message',
+                             data=f"Person {person_id}'s has {open_fracture_counts} open fractures without treatment",
+                             )
```

Comment:
```suggestion
            if self._number_of_times_this_event_has_run < self._maximum_number_times_event_should_run:
                self.sim.modules['RTI'].schedule_hsi_event_for_tomorrow(self)
            if pd.isnull(df.loc[person_id, 'rt_date_death_no_med']):
                    df.loc[person_id, 'rt_date_death_no_med'] = self.sim.date + DateOffset(days=7)
            logger.debug(key='rti_general_message',
                             data=f"Person {person_id}'s has {open_fracture_counts} open fractures without treatment",
                             )    
```

Reply:
@tbhallett I wanted your clarity on the `logger.debug()` upon reconsideration. Shouldn't `logger.debug()` be inside the `if pd.isnull()` condition. This implies if a person's `rt_date_death_no_med` is null, then we set the date and the logger message follows.  For instance, the code would look like this:

```suggestion
            if self._number_of_times_this_event_has_run < self._maximum_number_times_event_should_run:
                self.sim.modules['RTI'].schedule_hsi_event_for_tomorrow(self)
            if pd.isnull(df.loc[person_id, 'rt_date_death_no_med']):
                df.loc[person_id, 'rt_date_death_no_med'] = self.sim.date + DateOffset(days=7)
                logger.debug(key='rti_general_message',
                             data=f"Person {person_id}'s has {open_fracture_counts} open fractures without treatment",
                             )
```

Reply:
@tbhallett I wanted your clarity on the `logger.debug()` upon reconsideration. Shouldn't `logger.debug()` be inside the `if pd.isnull()` condition. This implies if a person's `rt_date_death_no_med` is null, then we set the date and the logger message follows.  For instance, the code would look like this:

```suggestion
            if self._number_of_times_this_event_has_run < self._maximum_number_times_event_should_run:
                self.sim.modules['RTI'].schedule_hsi_event_for_tomorrow(self)
            if pd.isnull(df.loc[person_id, 'rt_date_death_no_med']):
                df.loc[person_id, 'rt_date_death_no_med'] = self.sim.date + DateOffset(days=7)
                logger.debug(key='rti_general_message',
                             data=f"Person {person_id}'s has {open_fracture_counts} open fractures without treatment",
                             )
```

Reply:
Yes, you're right!!! Great spot.

I'll push commit to implement that change now

Reply:
Right! I also had at the master code. I guess we can leave it and let it resemble the master in that case